apps/picsa-tools/seasonal-calendar-tool/src/app/app-routing.module.ts

```diff
@@ -7,6 +7,16 @@ export const ROUTES_COMMON: Routes = [
     loadChildren: () => import('./pages/home/home.module').then((m) => m.HomeModule),
     title: 'Seasonal Calendar',
   },
+  {
+    path: 'create-calendar',
+    loadChildren: () => import('./pages/create-calendar/create-calendar.module').then((m) => m.CreateCalendarModule),
+    title: 'Create Calendar',
+  },
+  {
+    path: 'calendar-table',
```

Comment:
nit(non-blocking)
The urls might look a bit cleaner if instead of this path you used a route paramater, e.g. path: ':id' which will make the the id available on page load. E.g. picsa.app/seasonal-calendar/abcd123 will be able to load info for seasonal calendar with local id abcd123
https://angular.io/guide/router-reference#configuration

Reply:
Yes, Ill try to handle that issue aswell